pkg/cluster/cluster.go

```diff
@@ -58,6 +58,12 @@ func (c *Cluster) Start(ctx context.Context) (<-chan struct{}, error) {
 			return nil, err
 		}
 		c.setupEtcdProxy(ctx, etcdProxy)
+
+		// remove etcd member if it exists
+		if err := c.managedDB.RemoveSelf(ctx); err != nil {
+			return nil, err
```

Comment:
Is this the kind of error that should halt execution or maybe it makes more sense to WARN about it?

Reply:
yea I think it makes more sense to make a warning
I will fix